src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -147,21 +181,12 @@ public static ServerMessage joinGame(int gameId, WebSocketPlayer player) {
 
         // Joining the game was unsuccessful
         if (game == null) {
-            return ServerMessage.builder()
-                    .type(ServerMessage.MessageType.ERROR)
-                    .messagePath("join")
-                    .player(player)
-                    .build();
+            return ServerMessage.builder().type(ServerMessage.MessageType.ERROR).messagePath("join").player(player).build();
         } else {
             // Used when re-joining as the old player object gets reused
             WebSocketPlayer newPlayer = (WebSocketPlayer) game.getPlayers().stream().filter(player::equals).findFirst().orElseThrow();
 
-            return ServerMessage.builder()
-                    .type(ServerMessage.MessageType.SUCCESS)
-                    .messagePath("join")
-                    .jsonData(gson.toJson(game))
-                    .player(newPlayer)
-                    .build();
+            return ServerMessage.builder().type(ServerMessage.MessageType.SUCCESS).messagePath("join").jsonData(gson.toJson(game)).player(newPlayer).build();
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason